meta-resin-common/recipes-support/resin-vars/resin-vars/os-networkmanager.testconfig1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "os": {
+    "network": {
+      "wifi": {
+        "randomMacAddressScan": true
+      }
+    }
+  }
+}
```

Comment:
Are these used in the repo automatically? Or there for some manual testing when needed?

Reply:
Added routine to run the tests as well @ZubairLK

Reply:
Nice